sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/CategorizedEntity.java

```diff
@@ -43,6 +53,28 @@ public CategorizedEntity(String text, EntityCategory category, String subcategor
         this.category = category;
         this.subcategory = subcategory;
         this.confidenceScore = confidenceScore;
+        this.offset = 0;
+        this.length = 0;
+    }
+
+    /**
+     * Creates a {@link CategorizedEntity} model that describes entity.
+     *
+     * @param text The entity text as appears in the request.
+     * @param category The entity category, such as Person/Location/Org/SSN etc.
+     * @param subcategory The entity subcategory, such as Age/Year/TimeRange etc.
+     * @param offset The start position for the entity text.
+     * @param length The length for the entity text.
+     * @param confidenceScore A confidence score between 0 and 1 of the extracted entity.
+     */
+    public CategorizedEntity(String text, EntityCategory category, String subcategory, int offset, int length,
```

Comment:
should we keep the `offset` and `length` after `confidenceScore` so that someone that depended on previous constructor would use this by just appending properties?